### PR TITLE
Example: merge

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: Tasks/merge.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Merge two dictionaries
+
+merge_two_objects=(function merge_two_objects(object_1,object_2){
```
Comment (YaroslavKSE) [inline Tasks/merge.js:3]: We should better use :  const mergeTwoObjects = (firstObject, secondObject) =>

File: Tasks/merge.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    object_3 = [({})][0];
```
Comment (YaroslavKSE) [inline Tasks/merge.js:4]: To create empty object in Js we use: let mergedObject = {};

File: Tasks/merge.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+   for (attribute_name in object_1) {
```
Comment (YaroslavKSE) [inline Tasks/merge.js:5]: To better understanding I prefer to use: for (let object1Key in firstObject)

File: Tasks/merge.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+       (
+          object_3[attribute_name] == object_1[attribute_name]
+       );
```
Comment (YaroslavKSE) [inline Tasks/merge.js:8]: We don't need this lines here

File: Tasks/merge.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        object_3[attribute_name] = object_1[attribute_name]
```
Comment (YaroslavKSE) [inline Tasks/merge.js:9]: Improved naming: mergedObject[object1Key] = firstObject[object1Key]

File: Tasks/merge.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  }
+   for (attribute_name in object_2) {
```
Comment (YaroslavKSE) [inline Tasks/merge.js:11]: Ditto

File: Tasks/merge.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+      (object_3[attribute_name] == object_2[attribute_name])
```
Comment (YaroslavKSE) [inline Tasks/merge.js:12]: Don't need this line

File: Tasks/merge.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+     object_3[attribute_name] = object_2[attribute_name]
```
Comment (YaroslavKSE) [inline Tasks/merge.js:13]: Improved naming: mergedObject[object2Key] = secondObject[object2Key]

File: Tasks/merge.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+    }
+return object_1, object_2, object_3;
```
Comment (YaroslavKSE) [inline Tasks/merge.js:15]: We don't need to return three objects as we are creating new one: return mergedObject;

File: Tasks/merge.js [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+})
```
Comment (YaroslavKSE) [inline Tasks/merge.js:16]: We need to finish function's body using:  };

**Review by YaroslavKSE — CHANGES_REQUESTED** [review]
In overall I improved naming here, used Cammelcase insted of snakecase. Olso deleted some useless lines, used new structers insted of them.

**Comment by YaroslavKSE** [comment]
```Js
const mergeTwoObjects = (firstObject, secondObject) => {
    let mergedObject = {};
    for (let object1Key in firstObject) {
        mergedObject[object1Key] = firstObject[object1Key]
    }
    for (let object2Key in secondObject) {
        mergedObject[object2Key] = secondObject[object2Key]
    }
    return mergedObject;
};

const result = mergeTwoObjects({ a: 'uno', b: 'due' }, { c: 'tre' });
console.log(result)
```

**Comment by YaroslavKSE** [comment]
```py
def merge_two_dicts(first_dict, second_dict):
    merged_dict = {}
    merged_dict.update(first_dict), merged_dict.update(second_dict)
    return merged_dict


result = merge_two_dicts({'one': 1, 'two': 2}, {'name': 'Yaroslav'})
print(result)
```

**Comment by tshemsedinov** [comment]
Use const instead of let

**Comment by YaroslavKSE** [comment]
> Use const instead of let

Fixed

**Pull request closed without merging**